etl_manager/etl.py

```diff
@@ -112,16 +113,30 @@ def __init__(
 
         self.job_arguments = job_arguments
 
+        self.timeout_override_minutes = timeout_override_minutes
+
         self.github_py_resources = []
 
         # Set as default can change using standard getters and setters (except _job_run_id only has getter)
         self._job_run_id = None
         self.max_retries = 0
         self.max_concurrent_runs = 1
         self.allocated_capacity = 2
+
+
         self._glue_version = "1.0"
         self._python_version = "3"
 
+
+    @property
+    def timeout(self):
+        if self.timeout_override_minutes is None:
+            # 20 dollar limit, at 44 cents per worker per hour
+            return int(60 * (20 /(0.44 * self.allocated_capacity)))
```

Comment:
Shouldn't we move these numbers out to be parameters, in case we want to move the goalposts to $10 or $40 later?

Reply:
You can override these default values with

```
   g = GlueJob(
            "example/glue_jobs/simple_etl_job/",
            bucket="alpha-everyone",
            job_role="alpha_user_isichei",
            job_arguments={"--test_arg": "this is a test"},
            timeout_override_minutes = 2880
        )
```

So this is just providing a sensible failsafe for people who don't bother to think about cost....

I suppose we could set the `timeout_override_minutes` param to instead be `max job cost`.  That would make timeout number of minutes more obscure.

dunno...

Reply:
See new commit....dont know if fully addresses but the chosen constants are at least more transparent now